### ISSUE TITLE
Gen options: change recursive category

### ISSUE THREAD

**Issue by jbellik** [syntax-prosody-ot/main#11]
Create a version of Gen that has recursive prosodic words

**Comment by jbellik** [comment]
In the current implementation of GEN, the recursive category is always phi. Instead, which category should be recursive should be an argument to the function. For example: 
- if the argument is phi, then GEN would produce trees rooted in iota, where terminals are assumed to be w, and all intermediate levels are phis. 
- If the argument is w, then GEN would produce trees rooted in phi (?), where terminals are assumed to be feet, and all intermediate levels are w.
- if the argument is i, then GEN would produce trees rooted in iota (or utterance?), where terminals are assumed to be phis (? - for now), and all intermediate levels are iota.

In short, the current version needs to be abstracted away from the phi.
